### DISCUSSION TITLE
ISCE2 Sarpy Version

### DISCUSSION THREAD

**Discussion by JoshZakUrsa** [isce-framework/isce2#606] [Q&A]
I'm trying to get started with ISCE by running a RGZERO process, but I'm hitting errors where the code references "sicdmeta" instead of the correct syntax "sicd_meta". It seems that "sicdmeta" used to be correct for Sarpy<1.0 but we'd rather not use such an outdated version. Has anyone else seen something like this?
For more context I installed ISCE through condaforge and it didn't originally install Sarpy at all - I installed that manually and went with the latest version.

**Comment by piyushrpt** [comment]
SICD support within ISCE hasn't been updated in a long time, as it is a format that is not really used outside classified communities. Not much civilian data is distributed in that format. You may have to make additional changes using that sensor object, with newer versions of sarpy, as it is not actively maintained. The reader was developed with a very limited set of datasets.

**Reply by piyushrpt** [reply]
Also note that the logic in the reader pretty much assumes data is a stripmap collection and does not have the bells and whistles for spotlight data.
